save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/AwesomeBenchmarksView.kt

```diff
@@ -159,44 +175,49 @@ class AwesomeBenchmarksView : AbstractView<PropsWithChildren, AwesomeBenchmarksS
-                                                            "holder.js/32x32?theme=thumb&amp;bg=007bff&amp;fg=007bff&amp;size=1"
-                                                    attrs["src"] = "img/undraw_code_inspection_bdl7.svg"
-                                                    attrs["data-holder-rendered"] = "true"
-                                                    attrs["style"] = jso<CSSProperties> {
-                                                        width = 4.2.rem
-                                                    }.unsafeCast<Width>()
-                                                }
+                                            if (state.selectedMenuBench == benchmark.category) {
+                                                div("media text-muted ${if (i != 0) "pt-3" else ""}") {
+                                                    img(classes = "rounded mt-1") {
+                                                        attrs["data-src"] =
+                                                                "holder.js/32x32?theme=thumb&amp;bg=007bff&amp;fg=007bff&amp;size=1"
+                                                        attrs["src"] = "img/undraw_code_inspection_bdl7.svg"
+                                                        attrs["data-holder-rendered"] = "true"
+                                                        attrs["style"] = jso<CSSProperties> {
+                                                            width = 4.2.rem
+                                                        }.unsafeCast<Width>()
+                                                    }
 
-                                                p("media-body pb-3 mb-0 small lh-125 border-bottom border-gray text-left") {
-                                                    strong("d-block text-gray-dark") { +benchmark.name }
-                                                    +benchmark.description
-                                                    div("navbar-landing mt-2") {
-                                                        // FixMe: links should be limited with the length of the div
-                                                        benchmark.tags.split(",").map { " #$it " }.forEach {
-                                                            a("/#/awesome-benchmarks") {
-                                                                +it
+                                                    p("media-body pb-3 mb-0 small lh-125 border-bottom border-gray text-left") {
+                                                        strong("d-block text-gray-dark") { +benchmark.name }
+                                                        +benchmark.description
+                                                        div("navbar-landing mt-2") {
+                                                            // FixMe: links should be limited with the length of the div
```

Comment:
could you also fix this in your next PR? very long links should be limited by the length, because right now too long links break the alignment